### DISCUSSION TITLE
bagaimana cara mengurutkan data berupa karakter yg custom? :D

### DISCUSSION THREAD

**Discussion by KZF022329** [teamalgoritma/community#845] [Q&A] [answered]
Mohon bantuannya,

Bgm cara mengurutkan data tabel berupa karakter tapi sesuai urutan tertentu. soalnya, defaultnya menggunakan alphabetical order (CMIIW). misalnya:

SD, RA, MTS, TK, SMP, MI, SMA, MA, SMK
ingin diurutkan menjadi:
TK, RA, SD, MI, SMP, MTS, SMA, SMK, SMA

terima kasih banyak sebelumnya.

Salam

**Comment by muhrisman** [comment, marked as answer]
Untuk melakukan custome order seperti itu bisa pakai factor

**Reply by IrfanChairurrachman** [reply]
Sangat setuju dengan @muhrisman , 

Untuk melakukan pengurutan level kita bisa menggunakan parameter `levels`. Dengan contoh seperti berikut.
```
data$kolom <- factor(data$kolom, levels = c("TK", "RA", "SD", "dst"))
```

Semoga informasi ini dapat membantu. Bila ternyata informasi ini dapat menjawab pertanyaan Anda, Anda dapat “Mark as answer” jawaban ini.

**Reply by KZF022329** [reply]
many thx
